spec/ameba/rule/documentation/documentation_spec.cr

```diff
@@ -59,31 +59,31 @@ module Ameba::Rule::Documentation
         # bag
         macro bag
         end
-      CRYSTAL
+        CRYSTAL
     end
 
     it "fails if there is an undocumented public type" do
       expect_issue subject, <<-CRYSTAL
         class Foo
-      # ^^^^^^^^^ error: Missing documentation
+        # ^^^^^^^ error: Missing documentation
```

Comment:
does the `^^^...` error marker style work correctly for both cases? I was pretty sure there is a single way to write it.

Reply:
There is a single way to write it, these needed to be changed as they've been de-indented

Reply:
Why did the tests pass before if it was de-intended?

Reply:
The test worked before as it was indented 2 spaces, now it's indented 0

Reply:
Exactly, since the logic on error marker hasn't been changed, I assume it works with both cases, when it is indented 2 spaces or 0. Is that correct?

Reply:
I'm not sure what you're saying. I needed to change the error hint as I moved the heredoc end tag. IIRC the error annotations have to be an exact match, so there's only one error annotation form that can exist for a line.

Reply:
Sorry, i missed that the heredoc end tag was moved for this case.